### PR TITLE
feat: Voice activity tracking

### PR DESCRIPTION
**Pull request by zeroeightysix**

### CODE REVIEW

File: bot/src/main/kotlin/me/melijn/bot/database/model/VoiceJoins.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+@CreateTable
+object VoiceJoins : Table("voice_joins") {
+
+    val guildId = long("guild_id")
+    val channelId = long("channel_id")
+    val userId = long("user_id")
+    val timestamp = timestamp("timestamp")
+
+//    override val primaryKey: PrimaryKey = PrimaryKey(guildId, userId, timestamp)
```
Comment (ToxicMushroom) [inline bot/src/main/kotlin/me/melijn/bot/database/model/VoiceJoins.kt:15]: heh

File: bot/src/main/kotlin/me/melijn/bot/database/model/VoiceLeaves.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+@CreateTable
+object VoiceLeaves : Table("voice_leaves") {
+
+    val guildId = long("guild_id")
+    val channelId = long("channel_id")
+    val userId = long("user_id")
+    val timestamp = timestamp("timestamp")
+    val timeSpent = duration("time_spent").nullable()
+
+//    override val primaryKey: PrimaryKey = PrimaryKey(guildId, userId, timestamp)
```
Comment (ToxicMushroom) [inline bot/src/main/kotlin/me/melijn/bot/database/model/VoiceLeaves.kt:17]: heh

**Review by ToxicMushroom — CHANGES_REQUESTED** [review]
cleanup

File: bot/src/main/kotlin/me/melijn/bot/database/model/VoiceJoins.kt (hunk 1 above)
Comment (zeroeightysix) [reply bot/src/main/kotlin/me/melijn/bot/database/model/VoiceJoins.kt:15]: is for future use

**Comment by zeroeightysix** [comment]
review please

**Review by ToxicMushroom — APPROVED** [review]
すばらしい

**Pull request merged**